File: Estrutura_Repetição/C#/Ex1071/Main.cs
using System;

class MainClass {
    public static void Main(string[] args) {
        int x, y;
        x = Convert.ToInt32(Console.ReadLine());
        y = Convert.ToInt32(Console.ReadLine());

        int menor = Math.Min(x, y);
        int maior = Math.Max(x, y);

        if (menor % 2 == 0)
            menor += 1;
        else
            menor += 2;

        int soma = 0;
        for (int i = menor; i < maior; i += 2) {
            soma += i;
        }

        Console.WriteLine(soma);
    }
}

File: Estrutura_Repetição/Java/Ex1071/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int x = sc.nextInt();
        int y = sc.nextInt();

        int menor = Math.min(x, y);
        int maior = Math.max(x, y);

        if (menor % 2 == 0)
            menor += 1;
        else
            menor += 2;

        int soma = 0;
        for (int i = menor; i < maior; i += 2) {
            soma += i;
        }

        System.out.println(soma);

        sc.close();
    }
}